deployments/pulse3d/services/pulse3d/src/main.py

```diff
@@ -771,3 +770,23 @@ async def get_versions(request: Request):
     except Exception as e:
         logger.error(f"Failed to retrieve info of pulse3d versions: {repr(e)}")
         raise HTTPException(status_code=status.HTTP_500_INTERNAL_SERVER_ERROR)
+
+
+@app.get("/usage", response_model=UsageQuota)
+async def get_usage_quota(
+    request: Request, token=Depends(ProtectedAny(scope=PULSE3D_SCOPES)), service: str = "pulse3d"
```

Comment:
this is an example how to use Query params 
https://github.com/CuriBio/k8s_curi/blob/4d110d238be88e7135cf96729d5adfd4d42cefca/deployments/pulse3d/services/pulse3d/src/main.py#L100